heuristic/alns.py

```diff
@@ -29,6 +30,8 @@ def __init__(self, state, criterion, data, objective_weights, decay):
         self.current_solution = state
         self.best_solution = state
         self.best_legal_solution = state
+        self.initial_solution.write("Initial_solution")
```

Comment:
This can rather be handled in PR #168. See Sondre's comment regarding default saving.